src/main/java/de/tum/in/www1/artemis/service/dto/athena/TextExerciseDTO.java

```diff
@@ -0,0 +1,28 @@
+package de.tum.in.www1.artemis.service.dto.athena;
+
+import javax.validation.constraints.NotNull;
+
+import de.tum.in.www1.artemis.domain.TextExercise;
+
+/**
+ * A DTO representing a TextExercise, for transferring data to Athena
+ */
+public record TextExerciseDTO(long id, String title, Double maxPoints, double bonusPoints, String gradingInstructions, String problemStatement) {
```

Comment:
Why does Athena need the GradingInstructions?

Reply:
Athena CoFee does not require them, but in my modules I'm using the grading instructions for the LLM prompt.